docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -11,40 +11,45 @@ Additionally, it utilizes the [Channels toolkit](/api-docs/channels/guide/overvi
 
 ## Configure accounts
 
-<!-- theme: info -->
-> #### Store configuration
-> * Because we have a [store email address constraint](/api-docs/apps/guide/developer-portal#store-email-address-constraint?source=subscription-foundation) on draft and private apps, make sure that both your sandbox store and your Dev Portal account use the same email address. 
-> * The sandbox store must support multi-storefront sales.
+To configure your accounts you need to make sure the following requirements are met:
 
-1. To develop and test apps, you need a BigCommerce sandbox store. If you don't have one, [Create a Sandbox Store](/api-docs/partner/getting-started/create-a-sandbox-store?source=subscription-foundation).
-2. To register apps and create app API accounts, you need a BigCommerce Developer Portal account. If you don't have one, create a [Dev Portal account](https://devtools.bigcommerce.com/?source=subscription-foundation).
+1. Because there is a [store email address constraint](/api-docs/apps/guide/developer-portal#store-email-address-constraint?source=subscription-foundation) on draft and private apps, you need to make sure that both your sandbox store and your Dev Portal account use the same email address. 
+2. The sandbox store must support multi-storefront sales.
+3. To develop and test apps, you need a BigCommerce sandbox store. If you don't have one, [Create a Sandbox Store](/api-docs/partner/getting-started/create-a-sandbox-store?source=subscription-foundation).
+4. To register apps and create app API accounts, you need a BigCommerce Developer Portal account. If you don't have one, create a [Dev Portal account](https://devtools.bigcommerce.com/?source=subscription-foundation).
 
 ## Fork and install the source repository 
 
+As a first step you need to fork the source repository, clone it to your development, and install the dpackages for your app
+
+**To fork and install the source repository**
```

Comment:
this is the name of the section (which is also an anchor link). Why do we need a second bold heading?

Reply:
I've used this style in previous jobs to introduce a procedure. So it's not a heading but an intro style. We could leave it unbold